frame/nomination-pools/src/lib.rs

```diff
@@ -1258,34 +1381,32 @@ pub mod pallet {
 		pub fn unbond_other(
 			origin: OriginFor<T>,
 			delegator_account: T::AccountId,
+			mut unbonding_points: BalanceOf<T>,
 		) -> DispatchResult {
 			let caller = ensure_signed(origin)?;
 			let (mut delegator, mut bonded_pool, mut reward_pool) =
 				Self::get_delegator_with_pools(&delegator_account)?;
-
 			bonded_pool.ok_to_unbond_other_with(&caller, &delegator_account, &delegator)?;
 
-			// Claim the the payout prior to unbonding. Once the user is unbonding their points
-			// no longer exist in the bonded pool and thus they can no longer claim their payouts.
-			// It is not strictly necessary to claim the rewards, but we do it here for UX.
+			unbonding_points = unbonding_points.min(delegator.active_points());
+			// This is only for UX.
```

Comment:
```suggestion
			// Claim the the payout prior to unbonding. Once the user is unbonding their points
			// no longer exist in the bonded pool and thus they can no longer claim their payouts.
			// It is not strictly necessary to claim the rewards, but we do it here for UX.
```
opinionated, but i like the longer explanation :D

Reply:
Fair enough you win :D

Reply:
_your pallet, your rules_.